openmls/src/group/mls_group/config.rs

```diff
@@ -60,6 +60,10 @@ pub struct MlsGroupConfig {
     pub(crate) lifetime: Lifetime,
     /// Ciphersuite and protocol version
     pub(crate) crypto_config: CryptoConfig,
+    /// Extensions to be added to the own leaf node
+    pub(crate) leaf_extensions: Extensions,
+    /// Capabilities of the own leaf node
+    pub(crate) leaf_capabilities: Option<Capabilities>,
```

Comment:
Is there a reason we need these two to live in `MlsGroupConfig`? `MlsGroupConfig` is not just used to build the group, but is a fixed part of `MlsGroup`, where both extensions and capabilities are already stored in the owner's leaf.